app/src/main/res/layout/onlinefeedview_activity.xml

```diff
@@ -64,38 +66,39 @@
                         android:layout_width="0dp"
                         android:layout_height="wrap_content"
                         android:layout_alignParentTop="true"
-                        android:layout_marginBottom="16dp"
-                        android:layout_marginLeft="16dp"
-                        android:layout_marginStart="16dp"
-                        android:layout_marginTop="16dp"
-                        android:layout_marginRight="16dp"
-                        android:layout_marginEnd="16dp"
                         android:layout_alignParentEnd="true"
                         android:layout_alignParentRight="true"
-                        android:layout_toRightOf="@id/coverImage"
+                        android:layout_marginStart="16dp"
+                        android:layout_marginLeft="16dp"
+                        android:layout_marginTop="25dp"
+                        android:layout_marginEnd="26dp"
+                        android:layout_marginRight="26dp"
+                        android:layout_marginBottom="8dp"
                         android:layout_toEndOf="@id/coverImage"
+                        android:layout_toRightOf="@id/coverImage"
                         android:ellipsize="end"
+                        android:lineHeight="24dp"
                         android:maxLines="2"
                         android:shadowColor="@color/black"
                         android:shadowRadius="3"
                         android:textColor="@color/white"
-                        style="@style/AntennaPod.TextView.Heading"
+                        style="@style/AntennaPod.TextView.HeadingSemiBold"
                         tools:text="Podcast title" />
 
                     <TextView
                         android:id="@+id/author_label"
                         android:layout_width="0dip"
                         android:layout_height="wrap_content"
                         android:layout_below="@id/titleLabel"
-                        android:layout_marginBottom="16dp"
-                        android:layout_marginLeft="16dp"
-                        android:layout_marginStart="16dp"
-                        android:layout_marginRight="16dp"
-                        android:layout_marginEnd="16dp"
                         android:layout_alignParentEnd="true"
                         android:layout_alignParentRight="true"
-                        android:layout_toRightOf="@id/coverImage"
+                        android:layout_marginStart="16dp"
+                        android:layout_marginLeft="16dp"
+                        android:layout_marginEnd="16dp"
+                        android:layout_marginRight="16dp"
+                        android:layout_marginBottom="16dp"
                         android:layout_toEndOf="@id/coverImage"
+                        android:layout_toRightOf="@id/coverImage"
```

Comment:
These attributes are exactly the same as before. No re-ordering needed. Please restore the original order to keep the diff small and more manageable